packages/nameguard-react/src/components/Share/Share.tsx

```diff
@@ -77,13 +77,8 @@ export function Share({ name }: ShareProps) {
     <Fragment>
       <Tooltip
         trigger={
-          <IconButton
-            variant="ghost"
-            icon={
-              <ArrowUpTrayIcon className="text-black fill-current w-6 h-6" />
-            }
-            onClick={() => setIsOpen(true)}
-          >
+          <IconButton variant="ghost" onClick={() => setIsOpen(true)}>
```

Comment:
Yeah, but, the icon button needs the classes changed and padding fixed. So I will need to do another round on that specific component. Doing that should also remove the need for `!p-2` that @eduramme has added in places, because IconButton will have equal padding!